syndesis/plugins/org.fusesource.ide.syndesis.extension.ui/src/org/fusesource/ide/syndesis/extensions/ui/wizards/pages/SyndesisExtensionProjectWizardExtensionDetailsPage.java

```diff
@@ -56,28 +133,217 @@ public SyndesisExtensionProjectWizardExtensionDetailsPage() {
 	
 	@Override
 	public void createControl(Composite parent) {
+		DataBindingContext dbc = new DataBindingContext();
+		WizardPageSupport wps = WizardPageSupport.create(this, dbc);
+		wps.setValidationMessageProvider(dummyMessageProvider);
+		wizard = (SyndesisExtensionProjectWizard)getWizard();
+				
 		Composite container = new Composite(parent, SWT.NULL);
-		container.setLayout(new GridLayout(3, false));
+		container.setLayout(new GridLayout(4, false));
 
-		this.extensionIdText = createField(container, Messages.newProjectWizardExtensionDetailsPageExtensionIdLabel, null, Messages.newProjectWizardExtensionDetailsPageExtensionIdTooltip);
-		this.extensionVersionText = createField(container, Messages.newProjectWizardExtensionDetailsPageVersionLabel, null, Messages.newProjectWizardExtensionDetailsPageVersionTooltip);
-		this.extensionNameText = createField(container, Messages.newProjectWizardExtensionDetailsPageNameLabel, null, Messages.newProjectWizardExtensionDetailsPageNameTooltip);
-		this.extensionDescriptionText = createField(container, Messages.newProjectWizardExtensionDetailsPageDescriptionLabel, Messages.newProjectWizardExtensionDetailsPageOptionalDescriptionFieldHint, Messages.newProjectWizardExtensionDetailsPageDescriptionTooltip);
-		this.extensionTagsText = createField(container, Messages.newProjectWizardExtensionDetailsPageTagsLabel, Messages.newProjectWizardExtensionDetailsPageOptionalTagsFieldHint, Messages.newProjectWizardExtensionDetailsPageTagsTooltip);
+		createSyndesisVersionControls(container, dbc);
 		
-		setControl(container);
+		Label spacer = new Label(container, SWT.NONE);
```

Comment:
what about using Group instead of Label to create space and specific indent?

To check but I think that it is also better handled by accessibility tools.

For this iteration this is fine to let it as-is but will worth investigating/changing for further developments

Reply:
never did use groups for spacing afair. But we can for sure test it in a later iteration.